tracing/tests/macros.rs

```diff
@@ -346,6 +346,13 @@ fn enabled() {
     enabled!(Level::DEBUG);
     enabled!(target: "rando", Level::DEBUG);
     enabled!(target: "rando", Level::DEBUG, field);
+
+    span_enabled!(target: "rando", Level::DEBUG, field);
+    event_enabled!(target: "rando", Level::DEBUG, field);
+
+    // other cases
+    span_enabled!(Level::DEBUG);
+    span_enabled!(Level::DEBUG, foo, bar.baz, quux,);
```

Comment:
does this cover all possible cases of these macros? it looks like a few are missing here...